src/HouseofCat.RabbitMQ/Consumer/Consumer.cs

```diff
@@ -84,9 +84,9 @@ await _conLock
                     await SetChannelHostAsync().ConfigureAwait(false);
                     _shutdown = false;
                     _dataBuffer = Channel.CreateBounded<ReceivedData>(
-                        new BoundedChannelOptions(ConsumerOptions.BatchSize.Value)
+                        new BoundedChannelOptions(ConsumerOptions.BatchSize!.Value)
                         {
-                            FullMode = ConsumerOptions.BehaviorWhenFull.Value
+                            FullMode = ConsumerOptions.BehaviorWhenFull!.Value
```

Comment:
Nice, hadn't come across this mechanism before.

Note to self:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving